{"name": "testcaseadd3[【类型】输入中文，给出提示信息2]", "status": "passed", "steps": [{"name": "步骤一：4和字开始计算，预期应等于参数大小超出范围，打印<ExceptionInfo for raises contextmanager>", "status": "broken", "statusDetails": {"message": "TypeError: '>' not supported between instances of 'str' and 'int'\n", "trace": "  File \"D:\\zj\\PycharmProjects\\study202208\\practice\\study\\study0824\\testcase\\test_add.py\", line 48, in testcaseadd3\n    result = self.cal.add(x,y)\n  File \"D:\\zj\\PycharmProjects\\study202208\\script\\calculator.py\", line 4, in add\n    if a > 99 or a < -99 or b > 99 or b < -99:\n"}, "start": 1661497761858, "stop": 1661497761858}], "attachments": [{"name": "log", "source": "7dcf88e3-eada-4489-a39d-318c0661b092-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "x", "value": "4"}, {"name": "y", "value": "'字'"}, {"name": "expect", "value": "'参数大小超出范围'"}], "start": 1661497761857, "stop": 1661497761858, "uuid": "4775e2ef-a0cb-45a0-9b6c-3df1424f642c", "historyId": "f74a5785511cb0710fe0b991e4aa8757", "testCaseId": "2f8f937ca82ff540c8b5f46a0207716d", "fullName": "practice.study.study0824.testcase.test_add.Testadd#testcaseadd3", "labels": [{"name": "story", "value": "加法模块测试error"}, {"name": "tag", "value": "P2"}, {"name": "parentSuite", "value": "practice.study.study0824.testcase"}, {"name": "suite", "value": "test_add"}, {"name": "subSuite", "value": "Testadd"}, {"name": "host", "value": "LAPTOP-BEO6NH7O"}, {"name": "thread", "value": "33120-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "practice.study.study0824.testcase.test_add"}]}